src/napari_builtins/_skimage_data.py

```diff
@@ -32,8 +32,13 @@ def _load_skimage_data(name, **kwargs):
                 skimage.data.lily(),
                 {
                     'channel_axis': -1,
-                    'name': ['lily-R', 'lily-G', 'lily-W', 'lily-B'],
-                    'colormap': ['red', 'green', 'gray', 'blue'],
+                    'name': [
+                        'lily-cyan',
+                        'lily-magenta',
+                        'lily-yellow',
+                        'lily-white',
+                    ],
+                    'colormap': ['cyan', 'magenta', 'yellow', 'White'],
```

Comment:
```suggestion
                        'lily-magenta',
                        'lily-green',
                        'lily-yellow',
                        'lily-blue',
                    ],
                    'colormap': ['magenta', 'green', 'yellow', 'blue'],
```

I tried your various colormaps, and the last one was basically this! What's nice is this uses a string representation that will already be represented in our GUI, which is probably better for examples.

Reply:
Thanks @TimMonko .